git-sync/ssh.md

```diff
@@ -0,0 +1,62 @@
+# Using SSH with git-sync
+
+Git-sync supports using the SSH protocol for pulling git content.
+
+## Setup
+
+Write a config file for a Secret that holds your SSH private key, with the key mapped to "ssh".
```

Comment:
Can you also show a CLI example with `kubectl create secret` and `--from-file` ?  I think it will be more approachable.